docs/src/modules/components/AppTableOfContents.js

```diff
@@ -218,6 +219,55 @@ export default function AppTableOfContents(props) {
     <Nav aria-label={t('pageTOC')}>
       <NoSsr>
         <TableOfContentsBanner />
+        {showSurveyBanner && (
+          <Link
+            href={'https://www.surveymonkey.com/r/mui-developer-survey-2022?source=docs'}
+            underline="none"
+            sx={(theme) => ({
+              mb: 2,
+              display: 'flex',
+              flexDirection: 'column',
+              overflow: 'auto',
+              backgroundColor:
+                theme.palette.mode === 'dark'
+                  ? alpha(theme.palette.primary[900], 0.2)
+                  : alpha(theme.palette.grey[50], 0.4),
+              border: '1px solid',
+              borderColor:
+                theme.palette.mode === 'dark'
+                  ? theme.palette.primaryDark[700]
+                  : theme.palette.grey[200],
+              borderRadius: 1,
+              transitionProperty: 'all',
+              transitionTiming: 'cubic-bezier(0.4, 0, 0.2, 1)',
+              transitionDuration: '150ms',
+              '&:hover, &:focus-visible': {
+                borderColor:
+                  theme.palette.mode === 'dark'
+                    ? theme.palette.primaryDark[500]
+                    : theme.palette.primary[200],
+              },
+            })}
+          >
+            <Box sx={{ p: 1 }}>
+              <Typography component="span" variant="button" fontWeight="500" color="text.primary">
+                {'📫 MUI Developer Survey 2022 is live!'}
+              </Typography>
+              <Typography
+                component="span"
+                variant="caption"
+                fontWeight="normal"
+                color="text.secondary"
+                sx={{ mt: 0.5 }}
+              >
+                {/* eslint-disable-next-line material-ui/no-hardcoded-labels */}
+                {
+                  'Please take a few minutes to complete the survey. Let us know what we can improve and help us define the roadmap for 2023!'
```

Comment:
```suggestion
                  'Let us know what we can improve and help us define the roadmap for 2023!'
```

<img width="231" alt="Screen Shot 2022-12-10 at 12 22 38" src="https://user-images.githubusercontent.com/67129314/206862419-dd8cd633-1c17-44d6-ab33-23d29aeb6908.png">

Shall we try to shorten this text just a tiny bit? This TOC box description text should probably be limited to 1 to 2 rows max (this one being an exception as 2023 goes to the third). cc @samuelsycamore for any copy tips to shorten it even more!

Reply:
I think that it could be turned more into how it benefits the user:

```suggestion
Influence the future of MUI. Help define the roadmap for 2023!
```